java/java-tests/testSrc/com/intellij/psi/formatter/java/JavaFormatterAlignmentTest.java

```diff
@@ -49,6 +49,20 @@ public void testMethodAndChainedField() throws Exception {
     );
   }
 
+  public void testChainedMethodWithComments() throws Exception {
+    getSettings().ALIGN_MULTILINE_CHAINED_METHODS = true;
+    doMethodTest("AA.bb()\n" +
+                 ".cc() // comment after line\n" +
+                 ".dd()\n " +
+                 "        /* block comment on empty line */\n" +
```

Comment:
Why you've added this block comment line to test? 
Is it supposed for this comment to be indented with method chain start block?